doc/import-block-pv.md

```diff
@@ -0,0 +1,90 @@
+# How to import an image to a block raw PV User Guide
+The purpose of this document is to show how to import an image to a raw block PV.
+For now, this functionality only works for importing an image from a URL (http/s3) without authentication.
+
+## Prerequisites
+- You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+- Feature-Gate 'BlockVolume' is enabled.
+
+## Import an image to a Local raw block PV
+In case you do not import an image to a local PV, please skip to the next section (Import an image with DataVolume manifest).
+There are a few steps to follow, to be able to import an image to a local raw block PV.
+
+First, create a local PV with the volumeMode field set to 'Block':
+
+```bash
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: import-block-pv
+  annotations:
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node01
+  volumeMode: Block  # This is Block PV
+  storageClassName: local
+  capacity:
+    storage: 1Gi
+  local:
+    path: /dev/loop0   # This is the local path on the node where we import the image to
```

Comment:
below you create loop10. Maybe it should be consistent?

Reply:
done